ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -149,6 +149,11 @@ host_monitoring_cron:
   minute: "*/5"
   job: ops-runner -f -s 15 -n csnm.network.int /usr/bin/cron-send-network-metrics
 
+- name: report rpm versions daily
+  hour: "1"
+  minute: "0"
+  job: ops-runner -f -s 120 -n csdov.rpm.versions /usr/bin/cron-send-docker-oc-versions
```

Comment:
@zgalor You can't use the zaio to test this out?

My concern is that we're going to start having some checks in one and not the other. I think we need to keep these in sync.